cypress/integration/pages/search_spec.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference types="cypress" />
+export {};
+
+describe('Test Search Page', () => {
+  it('has a search form', () => {
+    cy.visit('/search');
+    cy.location('pathname').should('equal', '/search');
+    cy.get('form').find('input').type('world population');
+    cy.get('form').submit();
```

Comment:
I think we can have separate case for testing if the search form actually works.. eg, you type in and submit and get some results.

Reply:
Again, I'd test for expected result when you type "world population":

- query param
- it returns X number of results